modules/src/erc721/extensions/erc721metadata.rs

```diff
@@ -0,0 +1,50 @@
+use odra::{UnwrapOrRevert, Variable};
+use crate::erc721::extensions::erc721metadata::errors::Error;
+
+pub trait Erc721Metadata {
+    fn name(&self) -> String;
+    fn symbol(&self) -> String;
+    fn base_uri(&self) -> String;
+}
+
+#[odra::module]
+pub struct Metadata {
```

Comment:
This should contain Erc721 name in it.

Reply:
Changed the name of the trait to Erc721Metadata and the implementation to Erc721MetadataExtension.